src/v3/cli/commands/load.js

```diff
@@ -46,13 +54,49 @@ export default class PluginGraphCommand extends Command {
       flags: {
         "github-token": githubToken,
         "sourcecred-directory": basedir,
+        "max-old-space-size": maxOldSpaceSize,
         plugin,
       },
     } = this.parse(PluginGraphCommand);
-    loadPlugin({basedir, plugin, repoOwner, repoName, githubToken});
+    if (!plugin) {
+      loadAllPlugins({
+        basedir,
+        plugin,
+        repoOwner,
+        repoName,
+        githubToken,
+        maxOldSpaceSize,
+      });
+    } else {
+      loadPlugin({basedir, plugin, repoOwner, repoName, githubToken});
+    }
   }
 }
 
+function loadAllPlugins({repoOwner, repoName, githubToken, maxOldSpaceSize}) {
+  const tasks = [
+    ...pluginNames().map((pluginName) => ({
+      id: `load-${pluginName}`,
+      cmd: [
+        "node",
+        `--max_old_space_size=${maxOldSpaceSize}`,
+        "./bin/sourcecred.js",
+        "load",
+        repoOwner,
+        repoName,
+        "--plugin",
+        pluginName,
+        "--github-token",
+        githubToken,
```

Comment:
This passes the literal string `undefined` to the child process.

Reply:
Fixed.